bitnami/kafka/templates/statefulset.yaml

```
@@ -65,6 +65,9 @@ spec:
       {{- if .Values.hostAliases }}
       hostAliases: {{- include "common.tplvalues.render" (dict "value" .Values.hostAliases "context" $) | nindent 8 }}
       {{- end }}
+      {{- if .Values.schedulerName }}
+      schedulerName: {{ .Values.schedulerName }}
```

Should this filed be quoted for safety?
```suggestion
      schedulerName: {{ .Values.schedulerName | quote }}
```

---

applied